src/content/docs/controller.md

```diff
@@ -2,4 +2,218 @@
 title: Controller
 category: 02. Hardware
 page: 11
----
\ No newline at end of file
+---
+
+Controllers are the basis of how you control the V5 Brain remotely. You can upload programs and communicate with the competition switch using the controller. The controller has a screen, buttons, and joysticks that you can program to interact with the V5 Brain and its peripherals. The controller is also used to control the robot during the driver control period (opcontrol).
+
+During a match, one or two controllers can be used to control the robot. We'll refer to these controllers as the _primary controller_ and the _partner controller_. Having a partner controller is optional, but it can be useful for controlling additional functions of your robot. These two controllers are connected together with a smart cable.
+
+# Getting a reference to a `Controller`
```

Comment:
I think this should simply be getting a controller. You don't get a reference to a controller from peripherals.